.packit.yaml

```
@@ -0,0 +1,28 @@
+upstream_package_name: botocore
+downstream_package_name: python-botocore
+
+upstream_project_url: https://github.com/boto/botocore
+issue_repository: https://github.com/boto/botocore
+
+specfile_path: python-botocore.spec
+
+files_to_sync:
+  - .packit.yaml
+
+actions:
+  post-upstream-clone:
+    - wget https://src.fedoraproject.org/rpms/python-botocore/raw/main/f/python-botocore.spec
+  create-archive:
+    - python3 -m build --sdist --outdir ./fedora/
+    - bash -c "ls -1t ./fedora/*.tar.gz | head -n 1"
+
+srpm_build_deps:
+  - wget
+  - python3-build
+
+jobs:
+  - job: copr_build
+    trigger: pull_request
+    branch: develop
+    targets:
+      - fedora-rawhide
```

We can maybe add also the stable project with the merged state.

```suggestion
  - job: copr_build
    trigger: pull_request
    branch: develop
    targets:
      - fedora-rawhide
  - job: copr_build
    trigger: commit
    branch: develop
    targets:
      - fedora-rawhide
    preserve_project: true
```